src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -15,9 +15,11 @@
   [{:keys [theme type value account]}]
   (let [edited-account-data (assoc account type value)
         message             (case type
-                              :name  :t/edit-wallet-account-name-updated-message
+                              :name :t/edit-wallet-account-name-updated-message
                               :color :t/edit-wallet-account-colour-updated-message
                               :emoji :t/edit-wallet-account-emoji-updated-message
+                              :prod-preferred-chain-ids
+                              :t/edit-wallet-network-preferences-updated-message
                               nil)
```

Comment:
I'd suggest keep consistent the style of `case`:

```clojure
(case 
 :a :b 
 :c :d
 nil)
 
;; or

(case 
 :a
 :b
  
 :c
 :d
 
 nil)
```

but not mixed

Reply:
This is result of `lint-fix` command